Test/CGL2.xinp

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<simulation>
+
+  <!-- General - geometry definitions !-->
+  <geometry>
```

Comment:
use sets="true" and remove the topset please. less stuff to ponder over when brokenness hits us.

Reply:
and i cannot comment on above file, but nope. no VTF's please.